dm/tests/gbk/run.sh

```diff
@@ -67,6 +67,199 @@ function run() {
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
 	test_from_tidb
+
+	echo "prepare data for invalid connection test"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test1(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test2(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test3(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test4(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test5(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+	run_sql_source1 "CREATE TABLE IF NOT EXISTS gbk.invalid_conn_test6(i TINYINT, j INT UNIQUE KEY, k TINYINT, m TINYINT, n TINYINT)"
+
+	dmctl_stop_task $cur/conf/dm-task.yaml "--remove-meta"
```

Comment:
stop-task don't have "remove-meta", and no need to stop the task and source if we will add them again

Reply:
I tried to kill dm-worker without stopping the task, but the returned error said that dm-worker cannot be killed because there were still tasks running

Reply:
where do you see `returned error`? `kill` is a linux command, not related to DM's error

Reply:
Fixed. Have deleted the part of stopping task and source and creating them again, and can work correctly.